packages/badge/src/Badge.tsx

```diff
@@ -90,7 +91,7 @@ function Badge({
   variant = Variant.LightGray,
   className,
   ...rest
-}: BadgeProps & React.HTMLAttributes<HTMLDivElement>) {
+}: BadgeProps & HTMLElementProps<'div'>) {
```

Comment:
this seems to be the only place we leave out the `RefType` type argument. should this include an explicit `never` or should everywhere else omit it too?

Reply:
wrote it out explicitly